samples/application-conf/file.properties

```diff
@@ -17,18 +17,17 @@ dbb.scriptMapping = ZunitConfig.groovy :: **/*.bzucfg
 dbb.scannerMapping = ZUnitConfigScanner :: **/*.bzucfg
 
 #
-# General file level overwrites through DBB Build Properties 
-# isCICS = true :: **/cobol/member.cbl
-# isSQL = true :: **/cobol/member.cbl
-# isMQ = true :: **/cobol/member.cbl
-
-# 
-# Please check for available file property overwrites within samples/application-conf/README.md
+# Skip creating a load module for these programs as they will be statically linked to other programs
```

Comment:
@flpet-ibm Hi Flems, `samples/application-conf/file.properties` should not contain any specific configuration for MortgageApplication. 

I think you have combined the MortgageSample with the **Template** in sample/application-conf/